torch/utils/data/datapipes/iter/fileopener.py

```diff
@@ -28,21 +29,28 @@ def __init__(
             self,
             datapipe: Iterable[str],
             mode: str = 'r',
+            encoding: Optional[str] = None,
```

Comment:
Is `FileOpenerIterDataPipe` a new API introduced in 1.11?
 Because inserting new default argument in the middle of other arguments is not bakcward compatible (i.e. if I used to create it using `x= FileOpenerIterDataPipe("fooooo", r, 5)` it will stop working after this change

Reply:
Yes, this is newly introduced in 1.11

Reply:
And, in terms of BC breaking, this DataPipe is introduced in 1.11. And, this is tied to TorchData API, which was not officially released before PyTorch 1.11.